AzureTable/Microsoft.DataTransfer.AzureTable/Sink/ITableAPIBulkSinkAdapterConfiguration.cs

```diff
@@ -43,5 +43,11 @@ public interface ITableAPIBulkSinkAdapterConfiguration
         /// </summary>
         [Display(ResourceType = typeof(ConfigurationResources), Description = "BulkSink_MaxBatchSize")]
         int? MaxBatchSize { get; }
+
+        /// <summary>
+        /// Gets the value that indicates whether remote logging can be enabled in target.
+        /// </summary>
+        [Display(ResourceType = typeof(ConfigurationResources), Description = "BulkSink_RemoteLogging")]
+        bool RemoteLogging { get; }
```

Comment:
This should also be added to the UI in WPF project, as well as to generated command line there.

Reply:
N/A as code is in core now.